File: 26 Remove Duplicates from Sorted Array/Java/Solution.java
public class Solution {
    public int removeDuplicates(int[] nums) {
        int current = nums[0];
        int index = 1;
        for (int i = 1; i < nums.length; i++) {
            if (nums[i] != current) {
                nums[index] = nums[i];
                current = nums[index];
                index++;
            }
        }
        return index;
    }
}

File: 26 Remove Duplicates from Sorted Array/Python/Solution.py
from typing import List, Self


class Solution:
    def removeDuplicates(self: Self, nums: List[int]) -> int:
        current: int = nums[0]
        index: int = 1
        for i in range(1, len(nums)):
            if nums[i] != current:
                nums[index] = nums[i]
                current = nums[index]
                index += 1
        return index